src/NavigationService.js

```diff
@@ -0,0 +1,21 @@
+import { NavigationActions } from 'react-navigation';
```

Comment:
I'm not sure if it's that relevant but we should keep the initial file standard we have in the repo.